src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final int MAX_PROPERTY_VALUE = 50;
```

Comment:
```suggestion
    private static final int FIGURE_COUNT = 5;
    private static final int MAX_PROPERTY_VALUE = 50;
```

I think we can make this constant private if we dont use them outside of this class